fx/daemonsets/config

```diff
@@ -1,6 +1,6 @@
 HEADER='==========================================================================================================
 [F1]Explain [F2]Restart [F3]YAML [F4]Edit [F5]Descr search [F6]Trivy YAML scan [F7]Trivy image scan [F8]Delete
-Scrolling (SHIFT - up/down) || CTRL-/ (change view) || CTRL-R (refresh. omit -o wide) || Ctrl-L (-o wide)
+Scrolling (shift - up/down) || '$CTRL'-/ (change view) || '$CTRL'-R (refresh. omit -o wide) || Ctrl-L (-o wide)
```

Comment:
Since $CTRL is now a variable used in the fzf bind, it looked bad when lower case in the header, with SHIFT upper. Hope you don't mind that I lowered shift because of that (?)